pkg/multiTenancyPlugins/naming/nameScoping.go

```diff
@@ -248,3 +272,24 @@ func getContainerId(container *cluster.Container, tenantId string, containerRefe
 	}
 	return "", errors.New(CONTAINER_REFERENCE_NOT_FOR_CONTAINER_INFO)
 }
+
+// send request to next handler in record mode.  When it returns remove traces of tenant id in response
+func doReq(nameScoping *DefaultNameScopingImpl, command utils.CommandEnum, cluster cluster.Cluster, w http.ResponseWriter, r *http.Request, swarmHandler http.Handler) error {
```

Comment:
Ok.  I will move the encode/decode to/from apitypes.VolumeCreateRequest out of volumes.go to nameScoping.go.